README.md

```diff
@@ -1629,7 +1629,7 @@ Update Time, five active automations, webhooks.
 **[⬆️ Back to Top](#table-of-contents)**
 
 ## Analytics, Events and Statistics
-
+   * [Statsig](https://statsig.com) - All-in-one platform spanning across analytics, feature flagging, and A/B testing. Free for up to 1m metered events per month. 
```

Comment:
please maintain ordering and style - blank line after heading etc

Reply:
ok

Reply:
edited! thank you!